<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>961</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="scatter_vbox"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>530</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="longest_book_vbox"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="shortest_book_vbox"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>530</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="avg_page_number_vbox"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="page_number_vbox"/>
  </widget>
  <widget class="QPushButton" name="prev_screen_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza az előző elemzésekhez</string>
   </property>
  </widget>
  <widget class="QLabel" name="scatter_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="page_number_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="short_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="avg_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="long_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
